src/App/App.test.tsx

```diff
@@ -140,4 +167,22 @@ describe('App', () => {
     expect(screen.getAllByText('View song on Last FM')[0]).toBeInTheDocument()
     expect(screen.getAllByText('View artist on Last FM')[0]).toBeInTheDocument()
   });
+
+  test('should be able to star a in playlist details page', async () => {
+    (getGenres as jest.Mock).mockResolvedValue(mockGenres);
+    (getPlaylist as jest.Mock).mockResolvedValue(mockPlaylist1);
+    render(
+      <MemoryRouter>
+        <App />
+      </MemoryRouter>
+    );
+    const genreToClick: HTMLElement = await waitFor(() => screen.getByText('military western ska'))
+    userEvent.click(screen.getByText(genreToClick.innerHTML))
```

Comment:
This feels much more comprehensive from a user perspective on what they may experience going through the app.